src/chttpd/src/chttpd.erl

```diff
@@ -148,7 +151,12 @@ start_link(Name, Options) ->
     end.
 
 get_server_options(Module) ->
-    ServerOptsCfg = config:get(Module, "server_options", "[]"),
+    ServerOptsCfg =
+        case Module of
+            "chttpd" -> config:get(Module,
```

Comment:
strange indentation (we use `clause -> do_something` only if each clause fit on a single line).
```
case Module of
   "chttpd" ->
        config:get(Module, "server_options", ?DEFAULT_SERVER_OPTIONS);
    _ ->
        config:get(Module, "server_options", "[]")
end,
```

Reply:
Changed.
Thanks for the review, @iilyak